learn/cookbooks/running_production.mdx

```diff
@@ -3,106 +3,106 @@ title: Running Meilisearch in production — Meilisearch documentation
+Log into your server via SSH, update the list of available packages, and install `curl`:
 
-Once you are logged in into your machine via SSH, ensure your system and its dependencies are up-to-date before proceeding with the installation.
-
-```bash
-# Update the list of available packages and their versions
+```sh
 apt update
-# Install curl which is required to install Meilisearch in the next step
 apt install curl -y
-# Install Meilisearch latest version from the script
-curl -L https://install.meilisearch.com | sh
 ```
 
-The different Meilisearch installation options are detailed in [this guide](/learn/getting_started/quick_start#setup-and-installation).
+Using the latest version of a package is good security practice, especially in production environments.
 
-**There are many different ways to get Meilisearch running on your machine.** As an open-source project, you can always compile the latest stable release of Meilisearch from its source code to ensure the binary uses your architecture in the best possible way.
+Next, use `curl` to download and run the Meilisearch command-line installer:
 
-You can always check the latest Meilisearch stable version, and get Meilisearch for the Operating System of your choice, by visiting the following link:
+```sh
+# Install Meilisearch latest version from the script
+curl -L https://install.meilisearch.com | sh
+```
 
-**[Latest Meilisearch Stable Version](https://github.com/meilisearch/meilisearch/releases/latest)**
+The Meilisearch installer is a set of scripts that ensure you will get the correct binary for your system.
 
-Give the binary execute permission using:
+Give the binary the required execute permission:
 
-```bash
+```sh
 chmod +x meilisearch
 ```
 
-Meilisearch is finally installed and ready to use. To make it accessible from everywhere in your system, move the binary file into your system binaries folder:
+The last step is to make the binary accessible from anywhere in your system. Move the binary file into `/usr/local/bin`:
 
-```bash
-# Move the Meilisearch binary to your system binaries
+```sh
 mv ./meilisearch /usr/local/bin/
 ```
 
+Meilisearch is now ready to use. 
+
 ## Step 2: Create system user
 
-Running applications as root can introduce security flaws in your system. To prevent that from happening, create a dedicated system user for running Meilisearch:
+Running applications as root exposes you to unnecessary security risks. To prevent that, create a dedicated user for Meilisearch:
 
-```bash
+```sh
 useradd -d /var/lib/meilisearch -b /bin/false -m -r meilisearch
 ```
 
 ## Step 3: Create a configuration file
 
-Download default config to `/etc`:
+After installing Meilisearch and taking the first step towards keeping your data safe, you need to set up a basic configuration file.
+
+First, create the directories where Meilisearch will store its data:
+
+```bash
+mkdir /var/lib/meilisearch/data /var/lib/meilisearch/dumps /var/lib/meilisearch/snapshots
```

Comment:
Should we mention they should use the additional block storage? Because I'm not mistaken, they can't increase the size of their local node disk.

<img width="1299" alt="image" src="https://github.com/meilisearch/documentation/assets/4116980/7f140ef1-9641-42df-9dcc-d88bd895d2bb">

Reply:
That's a good point. I will try to drop in a reminder for users to check storage space and plan accordingly.